app/packages/operators/src/usePanelEvent.ts

```diff
@@ -33,19 +54,47 @@ export default function usePanelEvent() {
     const actualParams = {
       ...params,
       panel_id: panelId,
-      panel_state: currentPanelState ?? (panelState?.state || {}),
+      panel_state: currentPanelState ?? ((panelState as any)?.state || {}),
     };
 
     const eventCallback = (result, opts) => {
       decrement(panelId);
-      const msg =
-        result.errorMessage || result.error || "Failed to execute operation";
-      const computedMsg = `${msg} (operation: ${operator})`;
-      if (result?.error) {
-        notify({ msg: computedMsg, variant: "error" });
```

Comment:
Should we suppress the notify from `useOperatorExecutor` as well?

Reply:
Yep I've added a new option "skipErrorNotification" which is set to true when calling executeOperator via panel events.